qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/SelectItemsView.groovy

```diff
@@ -429,21 +425,29 @@ class SelectItemsView extends VerticalLayout{
             storageGrid.deselectAll()
             applyDataStorage.setEnabled(false)
         })
+
+        createOfferViewModel.productItems.addPropertyChangeListener({
+            if (createOfferViewModel.productItems) {
+                next.setEnabled(true)
+            } else {
+                next.setEnabled(false)
+            }
```

Comment:
So if we don't have product items we cannot proceed?
Isn't it better to throw an error here then because we know that we need to have the items in every case?

Reply:
hm, i thought to keep the user experience consistent, it would make sense to enable the next button only if product items are set

Reply:
We can leave it for now. Just wanted to make a remark. We can leave this to our users to decide how they prefer it.